libraries/radspreadprocessing/getting-started.md

```diff
@@ -87,6 +91,8 @@ In order to use the model of the __RadSpreadProcessing__ library in your project
 </tbody>
 </table>
 
+* When export to PDF format a document containing images different than **Jpeg** and **Jpeg2000** or **ImageQuality** different than High.
```

Comment:
*The Telerik.Documents.ImageUtils.dll assembly is needed when exporting to PDF format a document containing images different than **Jpeg** and **Jpeg2000** or **ImageQuality** different than High.*